util/importers/LDC97S62.py

```diff
@@ -364,11 +346,12 @@ def _maybe_split_dataset(filelist, target_dir):
             os.rename(wav_file, new_wav_file)
 
 
-def _read_data_set(work_dir, data_set, thread_count, batch_size, numcep, numcontext, limit=0):
+def _read_data_set(work_dir, data_set, thread_count, batch_size, numcep, numcontext, stride=1, offset=0, next_index=lambda i: i + 1, limit=0):
```

Comment:
I don't feel strongly about it and it could be useful for experiments so I guess it's fine to leave it in.